src/plugins/data/public/search/search_interceptor.ts

```diff
@@ -127,7 +128,7 @@ export class SearchInterceptor {
       ];
       const combinedSignal = getCombinedSignal(signals);
 
-      return search(request as any, { ...options, signal: combinedSignal }).pipe(
+      return search(request, { ...options, signal: combinedSignal }).pipe(
```

Comment:
`{ ...options, signal: combinedSignal }` looks a little bit strange.   Please go to  `ISearchOptions` interface declaration and see: 

```
export interface ISearchOptions {
  signal?: AbortSignal;
}
```
This interface contains only one field. Looks like it's enough to pass only `{ signal: combinedSignal }`